packages/lodestar/src/network/gossip/validator.ts

```diff
@@ -20,7 +20,8 @@ import {
 // Numbers from https://github.com/sigp/lighthouse/blob/b34a79dc0b02e04441ba01fd0f304d1e203d877d/beacon_node/network/src/beacon_processor/mod.rs#L69
 const gossipQueueOpts: {[K in GossipType]: {maxLength: number; type: QueueType}} = {
   [GossipType.beacon_block]: {maxLength: 1024, type: QueueType.FIFO},
-  [GossipType.beacon_aggregate_and_proof]: {maxLength: 1024, type: QueueType.LIFO},
+  // this is different from lighthouse's, there are more gossip aggregate_and_proof than gossip block
+  [GossipType.beacon_aggregate_and_proof]: {maxLength: 4096, type: QueueType.LIFO},
```

Comment:
what's the rationale for changing `maxLength` here?

Reply:
the issue itself is the queue length is not enough to handle SignedAggregateAndProof gossip topic, it's ~600 gossip messages per slot